packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -101,10 +101,15 @@ export class LocalSigningManager implements SigningManager {
    *
    * @param args.accounts - array of private keys
    */
-  public static async create(args: { accounts: PrivateKey[] }): Promise<LocalSigningManager> {
+  public static async create(args: {
```

Comment:
If we make this type into an interface, we could clean this up.

```
export interface LocalSigningManagerCreateArgs {
  accounts: PrivateKey[];
  curveType?: Curve;
}

 public static async create(args: LocalSigningManagerCreateArgs)
 
 private constructor(args: LocalSigningManagerCreateArgs)
```

This would mean moving the `sr25519` default back into the constructor, but we could write

`return new LocalSigningManager(args);`

which is nicer than repeating `accounts` multiple times